dali/driver/hid.py

```diff
@@ -207,14 +207,22 @@ async def send(self, command, in_transaction=False, exceptions=None):
             if not in_transaction:
                 self.transaction_lock.release()
 
-    async def run_sequence(self, seq, progress=None):
+    async def run_sequence(self, cmd_seq, progress=None):
```

Comment:
Suggestion: rename this to `execute(self, seq, progress=None)`

Reply:
Hmm.  `run_sequence` is possibly not the best name.  I'm not sure that `execute` is any clearer, however.  Perhaps something that mentions transactions?